### PR TITLE
Cron for send files to select less data and make more smaller queries

### PR DESCRIPTION
**Pull request by peterdietz**

### CODE REVIEW

File: classes/task/send_files.php [hunk 1]
```diff
@@ -36,7 +36,9 @@ public function plagiarism_vericite_send_files() {
 
             foreach ($dbfiles as $dbfile) {
                 try {
-                    $customdata = unserialize(base64_decode($dbfile->data));
+                    //Get data for this row
+                    $row = $DB->get_record('plagiarism_vericite_files', array('id' => $dbfile->id), 'id, cm, userid, identifier, data, status, attempts');
```
Comment (baholladay) [inline classes/task/send_files.php]: Shouldn't the first DB query be only getting the IDs? (i.e. line 26 $DB->get_records)

File: classes/task/send_files.php [hunk 2]
```diff
@@ -36,7 +36,9 @@ public function plagiarism_vericite_send_files() {
 
             foreach ($dbfiles as $dbfile) {
```
Comment (baholladay) [inline classes/task/send_files.php]: Can we rename this to something like dbFileId?

File: classes/task/send_files.php (hunk 1 above)
Comment (baholladay) [inline classes/task/send_files.php]: Can we rename "row" to "dbfile" so that this can be a shorter PR?

**Comment by peterdietz** [comment]
Pushed update to PR. Reduced the first query to only ID's, and changed variable names to reduce the amount of diff.

**Pull request merged**